src/sonar.ts

```diff
@@ -25,11 +35,35 @@ export async function retrieveSonarCloudResults(
     })
     .then((response) => {
       if (core.isDebug()) {
-        core.debug(
-          `Retrieved SonarCloud results: ${JSON.stringify(response.data)}`
+        core.info(
+          `Retrieved SonarCloud issues: ${JSON.stringify(response.data)}`
         );
       }
-      return response.data as SonarSearchResults;
+      return response.data as SonarSearchIssuesResult;
+    });
+}
+
+export async function retrieveSonarCloudHotspots(
```

Comment:
I understand why this is a separate function but it seems to share a lot of code with `retrieveSonarCloudIssues` 🤔